sympy/core/tests/test_sympify.py

```diff
@@ -502,3 +503,7 @@ def test_Range():
 
     assert sympify(builtin_range(10)) == Range(10)
     assert _sympify(builtin_range(10)) == Range(10)
+
+def test_sympify_set():
+    n = Symbol('n')
+    assert sympify({n}) == FiniteSet(n)
```

Comment:
If you use the set literal syntax here we won't be able to merge this until we remove Python 2.6 support in master (after the release).